k8s/pkg/registryserver/helpers.go

```diff
@@ -36,3 +36,14 @@ func mapNsmFromCustomResource(cdr *v1.NetworkServiceManager) *registry.NetworkSe
 		LastSeen: lastSeen,
 	}
 }
+
+func mapNseFromCustomResource(cdr *v1.NetworkServiceEndpoint, payload string) *registry.NetworkServiceEndpoint {
```

Comment:
Question: what `cdr` stands for?

Reply:
I meant Custom Defined Resource, but I think just 'cr' is more preferable. What do you think?

Reply:
Both crd and cr work for me.